sound/soc/sof/intel/cnl.c

```diff
@@ -411,3 +441,4 @@ const struct sof_intel_dsp_desc jsl_chip_info = {
 };
 EXPORT_SYMBOL_NS(jsl_chip_info, SND_SOC_SOF_INTEL_HDA_COMMON);
 MODULE_IMPORT_NS(SND_SOC_SOF_INTEL_CLIENT);
+MODULE_IMPORT_NS(SND_SOC_SOF_CLIENT);
```

Comment:
earlier patch?

Reply:
comment missed in updates? This doesn't seem to belong here?

Reply:
The earlier patch adds inte-client APIs that are common for all platforms and it is already imported in intel-client.c which will be a different module. Now will the probes client registration in cnl, we need this here as well.